### PR TITLE
[Cognito] Add new method adminConfirmSignUp

### PR DESCRIPTION
**Pull request by tyx**

### CODE REVIEW

File: src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    public function testAdminConfirmSignUpResponse(): void
+    {
+        // see https://docs.aws.amazon.com/cognitoidentityprovider/latest/APIReference/API_AdminConfirmSignUp.html
+        $response = new SimpleMockedResponse('{
+        }');
+
+        $client = new MockHttpClient($response);
+        $result = new AdminConfirmSignUpResponse(new Response($client->request('POST', 'http://localhost'), $client, new NullLogger()));
+        self::assertTrue($result->resolve());
```
Comment (tyx) [inline src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php:24]: Not sure how to test an empty response !

Comment (tyx) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php:24]: https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-cognito-idp-2016-04-18.html#adminconfirmsignup

File: src/Service/CognitoIdentityProvider/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+- Added operation `adminConfirmSignUp`
```
Comment (jderusse) [inline src/Service/CognitoIdentityProvider/CHANGELOG.md:5]: Missing `### Added` bloc

File: src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php (hunk 1 above)
Comment (Nyholm) [reply src/Service/CognitoIdentityProvider/tests/Unit/Result/AdminConfirmSignUpResponseTest.php:22]: Neither am I. Let's leave it like this for now

**Pull request merged**